apps/web/src/hooks/__tests__/use-recent-views.spec.tsx

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
I moved this test into a new file because I found that next-router-mock has some corner case which might not isolate with other test case